examples/mwaa/environment.yaml

```
@@ -0,0 +1,272 @@
+apiVersion: mwaa.aws.upbound.io/v1beta1
+kind: Environment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    name: example
+    dagS3Path: dags/
+    executionRoleArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    networkConfiguration:
+      - securityGroupIdRefs:
+          - name: example
+        subnetIdRefs:
+          - name: sample-subnet1
+          - name: sample-subnet2
+    region: us-west-1
+    sourceBucketArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: versioning-example
+
+---
+
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Role
+metadata:
+  annotations:
+    meta.upbound.io/example-id: iam/v1beta1/role
```

Please fix the `example-id`  annotations.

```suggestion
    meta.upbound.io/example-id: mwaa/v1beta1/environment
```